src/configs/recommended.ts

```diff
@@ -43,37 +38,21 @@ export const rules = {
     "comment-format": {
         options: ["check-space"],
     },
-    curly: true,
     "cyclomatic-complexity": false,
-    eofline: true,
     forin: true,
-    "import-spacing": true,
-    indent: {
-        options: ["spaces"],
-    },
-    "interface-over-type-literal": true,
     "jsdoc-format": true,
     "label-position": true,
     "max-classes-per-file": { options: 1 },
-    "max-line-length": { options: 120 },
-    "member-ordering": {
-        options: {
-            order: "statics-first",
-        },
-    },
     "new-parens": true,
     "no-angle-bracket-type-assertion": true,
     "no-any": false,
     "no-arg": true,
     "no-bitwise": true,
     "no-conditional-assignment": true,
-    "no-consecutive-blank-lines": true,
     "no-console": true,
     "no-construct": true,
     "no-debugger": true,
     "no-duplicate-super": true,
-    "no-empty": true,
```

Comment:
this isn't really a formatting rule, it's more about code style. I think we should keep it in the recommended ruleset